osbs/build/build_requestv2.py

```diff
@@ -0,0 +1,292 @@
+"""
+Copyright (c) 2018 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+from __future__ import print_function, absolute_import, unicode_literals
+
+import logging
+
+from osbs.build.build_request import BuildRequest
+from osbs.build.spec import BuildUserParams
+from osbs.constants import ISOLATED_RELEASE_FORMAT
+from osbs.exceptions import OsbsValidationException
+from osbs.utils import git_repo_humanish_part_from_uri, Labels
+
+logger = logging.getLogger(__name__)
+
+_CM_SECRETS = ['required_secrets', 'worker_token_secrets']
+
+
+class BuildRequestV2(BuildRequest):
+    """
+    Wraps logic for creating build inputs
+    """
+
+    def __init__(self, build_json_store, customize_conf=None):
+        """
+        :param build_json_store: str, path to directory with JSON build files
+        :param customize_conf: str, path to customize configuration JSON
+        """
+        super(BuildRequestV2, self).__init__(build_json_store, customize_conf)
```

Comment:
Does that need to be `customize_conf=customize_conf`? Otherwise it might be taken for `inner_template` due to being passed as a positional argument.

Come to think of it, I don't even know if passing `customize_conf` through the the superclass makes sense. @lcarva?

Reply:
Fixed but I'll amend it if lcarva thinks we don't need customize_conf.